lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs

```diff
@@ -295,27 +292,12 @@ unPersistMock bs
   where
     (header, body) = B8.splitAt (B8.length mockSealedTxMagic) bs
 
--- | Get the serialised transaction body and witnesses from a 'SealedTx'.
-getSerialisedTxParts :: SealedTx -> SerialisedTxParts
-getSerialisedTxParts (SealedTx _ (InAnyCardanoEra _ tx) _) = SerialisedTxParts
-    { serialisedTxBody = serialiseToCBOR $ Cardano.getTxBody tx
-    , serialisedTxWitnesses = serialiseToCBOR <$> Cardano.getTxWitnesses tx
-    }
-
```

Comment:
Do I understand it right that this function isn't used anywhere?

Reply:
Indeed